proxy/api/src/identity.rs

```diff
@@ -39,9 +39,38 @@ pub struct Identity {
 
 impl From<(PeerId, Person)> for Identity {
     fn from((peer_id, user): (PeerId, Person)) -> Self {
-        let urn = user.urn();
-        let handle = user.subject().name.to_string();
-        let ethereum = match user.payload().get_ext::<EthereumClaimExtV1>() {
+        let identity = RemoteIdentity::from(user);
+        let shareable_entity_identifier = Identifier {
+            handle: identity.metadata.handle.clone(),
+            peer_id,
+        };
+        Self {
+            peer_id,
+            urn: identity.urn,
+            shareable_entity_identifier,
+            metadata: identity.metadata,
+            avatar_fallback: identity.avatar_fallback,
+        }
+    }
+}
+
+/// The remote users personal identifying metadata and keys.
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RemoteIdentity {
+    /// The coco URN.
+    pub urn: Urn,
+    /// Bundle of user provided data.
+    pub metadata: Metadata,
+    /// Generated fallback avatar to be used if actual avatar url is missing or can't be loaded.
+    pub avatar_fallback: avatar::Avatar,
```

Comment:
Can we drop this for here and use the typescript library to generate this in the client? I’m ok if we do this as a follow-up when we remove avatars alltogether.

Reply:
I think that it should be a follow-up, these changes are way outside of this PR scope